ruby_on_rails/configure_ci.md

```diff
@@ -17,7 +17,7 @@ and the main branches already pushed and ready to be tested.
 renuo configure-semaphore
 ```
 
-The command will copy the necessary templates to `.semaphore` folder.
+The command will copy the necessary templates to `.semaphore` folder using the renuo-cli. These files need to be maintained on the [renuo-cli repository](https://github.com/renuo/renuo-cli/tree/main).
```

Comment:
This actually needs amending now. In the semaphore.yml there are commands generated that use the `--cache-folder` option which is now deprecated